src/components/chat/ChatInput.vue

```diff
@@ -84,8 +84,14 @@ export default {
     }
   },
   props: {
-    message: String,
-    stampAmount: Number
+    message: {
+      type: String,
+      default: () => ''
```

Comment:
Again, `required` might be more appropriate? From a type-perspective both ensure that `message: String`